doc/nrf/known_issues.rst

```diff
@@ -868,6 +868,14 @@ NCSDK-10196: DFU fails for some configurations with the quick session resume fea
 
   **Workaround:** Use the quick session resume feature only for configurations with the cellular network backend.
 
+Common Application Framework (CAF)
+==================================
+
+.. rst-class:: v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0
+
+Directed advertising (:kconfig:`CONFIG_CAF_BLE_ADV_DIRECT_ADV`) does not work properly.
```

Comment:
You need to add the related JIRA issue for this known issue. The update requires this part as mandatory, as mentioned in the beginning of the known_issues.rst raw file.